Formula/libvisual-0.4.rb

```diff
@@ -0,0 +1,23 @@
+class Libvisual04 < Formula
+  desc "Audio Visualization"
+  homepage "https://github.com/Libvisual/libvisual"
+  url "https://github.com/Libvisual/libvisual/releases/download/libvisual-0.4.2/libvisual-0.4.2.tar.gz"
+  sha256 "63085fd9835c42c9399ea6bb13a7ebd4b1547ace75c4595ce8e9759512bd998a"
+  license "LGPL-2.1-or-later"
+  # head "https://github.com/Libvisual/libvisual", branch: "0.4.x"
+
+  depends_on "pkg-config" => :build
+  depends_on "sdl12-compat"
+
+  def install
+    args = ["--prefix=#{prefix}"]
+    system "./configure", *args
+    system "make", "-j2"
+    system "make", "install"
+  end
+
+  test do
+    system "lv-tool-0.4", "--version"
+    system "lv-tool-0.4", "--help"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Can we start it, stop it, and make sure it didn't crash?

Reply:
@carlocab we can send sigint to it and check for exit code 130 but as a user I would expect that we do that in a virtual X11 like Xvfb then or something and probably only on Linux, so the users doesn't see windows popping up. Is that really doing that much for us here?

Reply:
@carlocab I have added a simple Xvfb based test now and fixed all `brew style` warnings. Ready for re-review in my eyes. Thanks in advance!